nucliadb_protos/nodereader.proto

```diff
@@ -254,7 +254,11 @@ message RelationEdgeFilter {
 }
 
 message RelationPrefixSearchRequest {
+    // Search for a single prefix
     string prefix = 1;
+    // Search for entities appearing on a query
+    optional string query = 3;
```

Comment:
What do you think of instead of adding the `query` field as an implicit oneof with `prefix`, rename `prefix` to query and add an enum or number to define how many words do we want to use (last 3, all of them...)